docs/routing/introduction.md

```diff
@@ -70,6 +70,8 @@ The example above uses multiple links. Each one maps a path (`href`) to a known
 
 Any `<Link />` in the viewport (initially or through scroll) will be prefetched by default (including the corresponding data) for pages using [Static Generation](/docs/basic-features/data-fetching/get-static-props.md). The corresponding data for [server-rendered](/docs/basic-features/data-fetching/get-server-side-props.md) routes is _not_ prefetched.
 
+Also note that when navigating to another page using `<Link />` even tough routing happens client side, the data for that page (via `GetServerSideProps`) _is_ fetched before navigating to the new route.
+
```

Comment:
```suggestion
```

I reworded and applied the suggestion by @styfle, thanks for making us aware! The existing sentence explains well enough how prefetching behaves, with a link to server-rendering, so it's a better fit to just expand that sentence IMO, too.